.circleci/config.yml

```diff
@@ -2,7 +2,7 @@
 executors:
   golang:
     docker:
-    - image: cimg/go:1.15
+    - image: cimg/go:1.17
```

Comment:
Want to bump all the way to 1.18?

Reply:
I saw an issue in the goreleaser repo about not working with 1.18 yet, and I did not want to make the can of worms even bigger :-P